### DISCUSSION TITLE
Including the right script file

### DISCUSSION THREAD

**Discussion by saadbouhya** [shipshapecode/shepherd#1448] [Q&A]
Hey,

is this the right tag that I should include in my index because it's not working, however the cdn works,

`<script src="shepherd-master/src/js/shepherd.js"></script>`

**Comment by RobbieTheWagner** [comment]
@saadbouhya you have to use the built version of shepherd. I would highly recommend using npm to install your dependencies and importing from there, but if that is not an option, all the built assets are on each release https://github.com/shipshapecode/shepherd/releases/tag/v8.3.0

**Comment by saadbouhya** [comment]
@rwwagner90 
Okay thanks, 
and could you please tell how to add the brightness focus effect as in the demo.

**Reply by RobbieTheWagner** [reply]
@saadbouhya you have to include the JS and CSS for the library. Then you can enable the modals with `useModalOverlay` please read the docs for all of the available options. https://shepherdjs.dev/docs/tutorial-02-usage.html

**Reply by saadbouhya** [reply]
Okay, thank you very much!
